recipes/podofo/all/conanfile.py

```diff
@@ -80,15 +80,7 @@ def requirements(self):
             self.requires("libunistring/0.9.10")
 
     def validate(self):
-        if self.info.settings.compiler.get_safe("cppstd") and Version(self.version) >= "0.9.7":
-            check_min_cppstd(self, 11)
-        if conan_version.major >= 2 and Version(self.version) < "0.9.7":
-            # FIXME: linter complains, but function is there
-            # https://docs.conan.io/2.0/reference/tools/build.html?highlight=check_min_cppstd#conan-tools-build-check-max-cppstd
-            import sys
-            check_max_cppstd = getattr(sys.modules['conan.tools.build'], 'check_max_cppstd')
-            # INFO: error: no template named 'auto_ptr' in namespace 'std'. Removed in C++17.
-            check_max_cppstd(self, 14)
+        check_min_cppstd(self, 11)
```

Comment:
Do we not need to keep the `check_max_cppstd(self, 14)` for versiones older than `0.9.7`?

Reply:
Not needed! The only supported version in CCI right now is `0.9.7`:
https://github.com/conan-io/conan-center-index/blob/master/recipes/podofo/config.yml?plain=1#L1